lms/static/sass/discussion/_discussion.scss

```diff
@@ -297,8 +281,8 @@ body.discussion {
     width: 100%;
     height: 31px;
     padding: 0 ($baseline/2);
-    @include box-sizing(border-box);
-    border: 1px solid $gray-l2;
+    box-sizing: border-box;
```

Comment:
I think the `@include` uses Bourbon and automatically spits out different rules for various browsers.